cogeo_mosaic/backends/base.py

```diff
@@ -64,3 +69,83 @@ def write(self):
     @abc.abstractmethod
     def update(self):
         """Update existing MosaicJSON on backend."""
+
+    def _update_quadkey(self, quadkey: str, dataset: List[str]):
+        """Update quadkey list."""
+        self.mosaic_def.tiles[quadkey] = dataset
+
+    def _update_metadata(self, bounds: List[float], version: str):
+        """Update bounds and center."""
+        self.mosaic_def.version = version
+        self.mosaic_def.bounds = bounds
+        self.mosaic_def.center = (
+            (bounds[0] + bounds[2]) / 2,
+            (bounds[1] + bounds[3]) / 2,
+            self.mosaic_def.minzoom,
+        )
```

Comment:
@kylebarron is a Tuple ok for dynamoDB ?

Reply:
I think so, but you need to coerce the numeric values to `Decimal`.

Reply:
Definitely something you need to test.

Reply:
FYI, because we do `meta = json.loads(json.dumps(self.metadata), parse_float=Decimal)`  before uploading the metadata to dynamDB, the `bounds` will be stored as a `[ ]` but when loading to the mosaicJSON it will be asigned to Tuple as expected